.grit/patterns/moment-to-date-fns.grit

```diff
@@ -0,0 +1,781 @@
+language js
+
+// A call to `add`/`sub` method of a `momentjs` date object.
+pattern moment_arith($date, $method, $count, $specifier) {
+  `$date.$method($count, $specifier)` as $arith_exp where {
+    $method <: or { `add` => `addDate`, `subtract` => `subDate` },
+    if ($method <: `add`) {
+      $method <: ensure_import_from(source=`"date-fns/add"`),
+    } else {
+      $sub = `sub`,
+      $sub <: ensure_import_from(source=`"date-fns/sub"`),
```

Comment:
```suggestion
      $method <: ensure_import_from(source=`"date-fns/sub"`),
```

Reply:
`$method` here is bound to 'subtract', not `sub`.
moment().subtract() is equivalent to datefns.sub().